kubemarine/resources/schemas/cluster.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "nodes": {
+      "type": "array",
+      "description": "Describe each node of the cluster.",
+      "minItems": 1,
+      "items": {
+        "$ref": "definitions/node.json"
+      }
+    },
+    "node_defaults": {
+      "$ref": "definitions/node_defaults.json"
+    },
+    "cluster_name": {
+      "type": "string",
+      "description": "Address of the cluster on which the Control Plane Endpoint is raised, and used in the calculated parameters."
+    },
+    "control_plain": {
+      "type": "object",
+      "description": "Specifies which addresses are to be available for Kubernetes.",
+      "properties": {
+        "internal": {
+          "type": "string",
+          "description": "Internal network address for the connection. It is used for all internal kubeapi traffic."
```

Comment:
Hi @shmo1218,
In this and below descriptions I personally prefer `To be used` or at least `It is to be used`, as the parameters manage how Kubemarine will configure and use the addresses, not how it is configured/used now.
But anyway, these descriptions are taken as-is from https://github.com/Netcracker/KubeMarine/blob/feature/yaml_schema/documentation/Installation.md#control_plain so both places should be the same.

Reply:
Hi, @ilia1243,

I have updated the statements as "It is to be..."